vertx-db2-client/src/test/java/io/vertx/db2client/DB2TestBase.java

```diff
@@ -21,12 +26,17 @@
 	protected Vertx vertx;
 	protected Connector<SqlConnection> connector;
 	protected DB2ConnectOptions options;
+	
+	@Rule
+	public TestName testName = new TestName();
 
 	@Before
 	public void setUp(TestContext ctx) throws Exception {
+		System.out.println(">>> BEGIN " + testName.getMethodName());
 		vertx = Vertx.vertx();
 		initConnector();
-		cleanTestTable(ctx);
+		for (String table : tablesToClean())
+			cleanTestTable(ctx, table);
 	}
```

Comment:
tablesToClean() always returns an empty list each time, so won't that mean cleanTestTable() will never be run?

Reply:
by default yes, but if a test class wants to have tables cleaned they can override the `tablesToClean()` method like this:
```java
	protected List<String> tablesToClean() {
		return Collections.singletonList("someTable");
	}
```

Reply:
Ok, that makes sense.  Thanks for the explanation.